src/utils/index.ts

```diff
@@ -0,0 +1,50 @@
+import * as jwt from 'jsonwebtoken';
+import nodemailer = require('nodemailer');
+import { jwtSecret, mail, url } from '../config/config';
+import { codeErrors } from '../config/config';
+import { SecureError } from './errors';
+
+const transport = nodemailer.createTransport(mail);
+
+export const sendMail = (email: string, name: string): void => {
+  const token: any = getToken({ email });
+  const html = generateEmail(name, email, token);
+  transport.sendMail(
+    {
+      from: 'Car Fdinder',
+      html,
+      subject: 'Confirma registration',
+      to: email
+    },
+    err => {
+      if (err) {
+        throw err;
+      }
+    }
+  );
+};
+
+const generateEmail = (name: string, email: string, token: string): string => {
+  return `
+  <div style="text-align: center;">
```

Comment:
We could use any kind of a template engine. E.g. handlebars/pug whatever or something different here.
Although let's keep it like this for now.

Reply:
Is it good practice to use template engine for generating a little piece of code?We will not use it anymore.
But if you want we can use pug =) Pug is cool :+1:

Reply:
Yes, it is a good practice as you might need more email in future.
Personally, I hate pug because of the syntax.

We can keep it as is for now.